app/middlewares/validators/users.js

```diff
@@ -10,3 +12,12 @@ exports.checkExistingEmail = async (req, _, next) => {
   }
   return next();
 };
+
+exports.findAndDecryptPassword = async (req, _, next) => {
```

Comment:
This is totally fine but, as this is on of the main parts of the login bussiness logic, I think it would make more sense to pu this logic in the controller or an interactor called in the controller